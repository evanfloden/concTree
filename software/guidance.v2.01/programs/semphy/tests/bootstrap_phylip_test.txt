(A,(B,(H,(D,(J,(((G,E),(F,I)),C))))));
(A,(B,(D,((J,H),(((G,E),(F,I)),C)))));
(A,(B,(D,(H,(J,(((G,E),(F,I)),C))))));
(A,(B,(E,(G,((F,I),((J,(H,D)),C))))));
(A,(B,(E,(G,((F,I),(((J,H),D),C))))));
(A,(B,(E,((F,I),(G,((J,(H,D)),C))))));
(A,(B,(E,((F,I),(G,(((J,H),D),C))))));
(A,(B,(E,((G,(F,I)),((J,(H,D)),C)))));
(A,(B,(E,((G,(F,I)),(((J,H),D),C)))));
