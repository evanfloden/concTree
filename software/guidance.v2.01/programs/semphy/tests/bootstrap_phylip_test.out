creating a bootstrap object from a file
 first constractor
9	[1:1]   0  | 1 2 3 4 5 6 7 8 9 
9	[2:1]   0 1  | 2 3 4 5 6 7 8 9 
1	[3:1]   0 1 2  | 3 4 5 6 7 8 9 
2	[4:1]   0 1 2 3  | 4 5 6 7 8 9 
3	[5:1]   0 1 2 3 4  | 5 6 7 8 9 
9	[2:1]   0 1 2 3 4 5 6 9  | 7 8 
2	[3:1]   0 1 2 3 4 6 9  | 5 7 8 
3	[2:1]   0 1 2 3 4 7 8 9  | 5 6 
3	[4:1]   0 1 2 3 4 9  | 5 6 7 8 
2	[3:1]   0 1 3  | 2 4 5 6 7 8 9 
4	[2:1]   0 1 3 5 6 7 8 9  | 2 4 
3	[2:1]   0 1 4 5 6 7 8 9  | 2 3 
2	[4:1]   0 1 5 6  | 2 3 4 7 8 9 
6	[4:1]   0 1 5 6 7 8  | 2 3 4 9 
6	[3:1]   0 1 5 6 7 8 9  | 2 3 4 
6	[3:1]   0 1 6  | 2 3 4 5 7 8 9 
2	[5:1]   0 1 6 7 8  | 2 3 4 5 9 

 secound constractor
9	[1:1]   0  | 1 2 3 4 5 6 7 8 9 
9	[2:1]   0 1  | 2 3 4 5 6 7 8 9 
1	[3:1]   0 1 2  | 3 4 5 6 7 8 9 
2	[4:1]   0 1 2 3  | 4 5 6 7 8 9 
3	[5:1]   0 1 2 3 4  | 5 6 7 8 9 
9	[2:1]   0 1 2 3 4 5 6 9  | 7 8 
2	[3:1]   0 1 2 3 4 6 9  | 5 7 8 
3	[2:1]   0 1 2 3 4 7 8 9  | 5 6 
3	[4:1]   0 1 2 3 4 9  | 5 6 7 8 
2	[3:1]   0 1 3  | 2 4 5 6 7 8 9 
4	[2:1]   0 1 3 5 6 7 8 9  | 2 4 
3	[2:1]   0 1 4 5 6 7 8 9  | 2 3 
2	[4:1]   0 1 5 6  | 2 3 4 7 8 9 
6	[4:1]   0 1 5 6 7 8  | 2 3 4 9 
6	[3:1]   0 1 5 6 7 8 9  | 2 3 4 
6	[3:1]   0 1 6  | 2 3 4 5 7 8 9 
2	[5:1]   0 1 6 7 8  | 2 3 4 5 9 

geing whights from a tree
 0 1 1 0 1 0 0.111111 0 0.222222 0 0.333333 0.333333 0.333333 0 0 1 0 0 0
checkout the support of a tree
(A,(B,(H,(D,(J,(((G,E)[0.333333],(F,I)[1])[0.333333],C)[0.333333])[0.222222])[0.111111])[1])[1]);

remove the first tree from the list, and use is as bases for additional computation
9	[1:1]   0  | 1 2 3 4 5 6 7 8 9 
9	[2:1]   0 1  | 2 3 4 5 6 7 8 9 
3	[3:1]   0 1 2  | 3 4 5 6 7 8 9 
3	[5:1]   0 1 2 3 4  | 5 6 7 8 9 
9	[2:1]   0 1 2 3 4 5 6 9  | 7 8 
2	[3:1]   0 1 2 3 4 6 9  | 5 7 8 
3	[2:1]   0 1 2 3 4 7 8 9  | 5 6 
3	[4:1]   0 1 2 3 4 9  | 5 6 7 8 
1	[4:1]   0 1 2 4  | 3 5 6 7 8 9 
5	[2:1]   0 1 2 5 6 7 8 9  | 3 4 
3	[2:1]   0 1 3 5 6 7 8 9  | 2 4 
2	[4:1]   0 1 5 6  | 2 3 4 7 8 9 
6	[4:1]   0 1 5 6 7 8  | 2 3 4 9 
6	[3:1]   0 1 5 6 7 8 9  | 2 3 4 
6	[3:1]   0 1 6  | 2 3 4 5 7 8 9 
2	[5:1]   0 1 6 7 8  | 2 3 4 5 9 
 0 1 1 0 1 0 0 0 0.111111 0 0.333333 0.333333 0.333333 0 0 1 0 0 0
checkout the support of the removed tree
(A,(B,(H,(D,(J,(((G,E)[0.333333],(F,I)[1])[0.333333],C)[0.333333])[0.111111]))[1])[1]);

compatability 0.5 = Majority Rule
<0,1>:1
<2,16>:1
<2,17>:1
<4,0>:1
<4,3>:1
<6,4>:0.666667
<6,14>:0.666667
<8,5>:0.666667
<8,7>:0.666667
<8,9>:0.666667
<10,8>:0.666667
<10,18>:0.666667
<11,2>:0
<11,6>:0
<11,10>:0
<11,13>:0

from the consensus
((B:1.000000,(E:1.000000,((F:1.000000,I:1.000000):1.000000,((H:1.000000,D:1.000000,J:1.000000):1.000000,C:1.000000):1.000000,G:1.000000):1.000000):1.000000):1.000000,A:1.000000);
with support
((B,(E,((F,I)[1.000000],((H,D,J)[0.666667],C)[0.666667],G))[0.666667])[1.000000],A)[1.000000];

compatability 0.1
<0,1>:1.000000
<2,16>:1.000000
<2,17>:1.000000
<4,0>:1.000000
<4,3>:1.000000
<6,5>:0.444444
<6,9>:0.444444
<8,4>:0.666667
<8,14>:0.666667
<10,6>:0.666667
<10,7>:0.666667
<11,10>:0.666667
<11,18>:0.666667
<12,11>:0.222222
<12,13>:0.222222
<15,2>:0.000000
<15,8>:0.000000
<15,12>:0.000000

from the consensus
((B:1.000000,(E:1.000000,((F:1.000000,I:1.000000):1.000000,((((H:1.000000,J:1.000000):1.000000,D:1.000000):1.000000,C:1.000000):1.000000,G:1.000000):1.000000):1.000000):1.000000):1.000000,A:1.000000);
with support
((B,(E,((F,I)[1.000000],((((H,J)[0.444444],D)[0.666667],C)[0.666667],G)[0.222222]))[0.666667])[1.000000],A)[1.000000];
compatability 0.9
<0,1>:1.000000
<2,16>:1.000000
<2,17>:1.000000
<4,0>:1.000000
<4,3>:1.000000
<6,2>:0.000000
<6,4>:0.000000
<6,5>:0.000000
<6,7>:0.000000
<6,9>:0.000000
<6,13>:0.000000
<6,14>:0.000000
<6,18>:0.000000

from the consensus
((B:1.000000,((F:1.000000,I:1.000000):1.000000,H:1.000000,D:1.000000,J:1.000000,G:1.000000,E:1.000000,C:1.000000):1.000000):1.000000,A:1.000000);
with support
((B,((F,I)[1.000000],H,D,J,G,E,C))[1.000000],A)[1.000000];
strict consensus
<0,1>:1.000000
<2,16>:1.000000
<2,17>:1.000000
<4,0>:1.000000
<4,3>:1.000000
<6,2>:0.000000
<6,4>:0.000000
<6,5>:0.000000
<6,7>:0.000000
<6,9>:0.000000
<6,13>:0.000000
<6,14>:0.000000
<6,18>:0.000000

from the consensus
((B:1.000000,((F:1.000000,I:1.000000):1.000000,H:1.000000,D:1.000000,J:1.000000,G:1.000000,E:1.000000,C:1.000000):1.000000):1.000000,A:1.000000);
with support
((B,((F,I)[1.000000],H,D,J,G,E,C))[1.000000],A)[1.000000];
